### PR TITLE
Add portainer template for recipes

### PR DESCRIPTION
**Pull request by dancn**

### CODE REVIEW

**Comment by taliaga** [comment]
Do you think instead of a single template file with all of them we could still keep the by-GE division and have for example the converter collect them all by exploring the subfolders?

**Comment by dancn** [comment]
@taliaga this push splits the templates as suggested!

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+PORTAINER_TEMPLATES_YAML = $(shell find -name 'portainer-template.yaml' | sort)
+
+portainer/templates.json: $(PORTAINER_TEMPLATES_YAML)
+# Will wait forever miserably if the requisites are empty
```
Comment (taliaga) [inline Makefile:4]: which requirements? :P

File: utilities/yaml2json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import sys
+
+fin = sys.stdin
+fout = sys.stdout
+
+with fin as hin:
+    with fout as wout:
+        data = yaml.safe_load(hin)
+        json.dump(data, wout, indent=4)
+        # json.dump(wout, data)
```
Comment (taliaga) [inline utilities/yaml2json]: leftover

**Review by taliaga — COMMENTED** [review]
Much better now! next step then (not for this PR) will be to get those .yaml files automatically generated or somehow merged with the docker-compose files, to keep a single source of definition/documentation.

File: Makefile (hunk 1 above)
Comment (taliaga) [reply Makefile:4]: just fyi, here, in the same fashion, we'll add later the step to fetch recipes stored externally. In my branch is a simple  `build.sh`

Comment (dancn) [reply Makefile:4]: Hi @taliaga 
if there are no requisites the grep in next lines will wait forever because no arguments, really and edge case...

**Pull request merged**